[train-141][stage] packaged code is not pointed at the CDN

With v1.141.1 in stage, there are dozens of errors in the server tests like:

```
#https get https://accounts.stage.mozaws.net/authorization (18s)
[19:15:06]
AssertionError: https://accounts.stage.mozaws.net/bundle-67f46f72f9c77b5c06ebadc05f347ea498b9baec/head.bundle.js should not have CORS headers: expected true to be falsy
  at <tests/server/helpers/routesHelpers.js:205:18>
```

I think the packaging-for-CDN isn't right in 1.141. If I put back v1.140.3 in stage, for the top-level '/', I see, for one example, this difference:
```
v1.140.3  <script src="https://accounts-static-2.stage.mozaws.net/bundle-59c9857e777d19e6884edac7be1b399693fb21b9/head.bundle.js" ...

v1.141.1  <script src=/bundle-67f46f72f9c77b5c06ebadc05f347ea498b9baec/head.bundle.js ...
```
So, in v1.141.1, sub-resources are coming from the origin, not the CDN.

By the way, when I put back v1.140.3, I used the same version of cloudops-deployment, so I don't think the deployment pipeline is the root cause (unless it's some new config that has been added in 141).

--- a/packages/fxa-content-server/.prettierignore
+++ b/packages/fxa-content-server/.prettierignore
@@ -18,3 +18,8 @@ app/scripts/vendor/*
 tests/teamcity/*
 tests/tools/certs/*
 server/templates/pages/dist/*
+# The way prettier reformats the HTML causes
+# grunt-cdn to break & JS files are served
+# from nginx rather than the CDN
+server/templates/pages/src/index.html
+server/templates/pages/src/update_firefox.html

--- a/packages/fxa-content-server/server/templates/pages/src/index.html
+++ b/packages/fxa-content-server/server/templates/pages/src/index.html
@@ -24,10 +24,7 @@
         <link rel="stylesheet" href="/styles/main.css" />
         <!-- endbuild -->
 
-        <script
-            type="text/javascript"
-            src="{{ bundlePath }}/head.bundle.js"
-        ></script>
+        <script type="text/javascript" src="{{ bundlePath }}/head.bundle.js"></script>
     </head>
     <body
         data-flow-id="{{flowId}}"
@@ -57,13 +54,7 @@
             {{#t}}Firefox Accounts requires JavaScript.{{/t}}
         </noscript>
 
-        <script
-            type="text/javascript"
-            src="{{ bundlePath }}/appDependencies.bundle.js"
-        ></script>
-        <script
-            type="text/javascript"
-            src="{{ bundlePath }}/app.bundle.js"
-        ></script>
+        <script type="text/javascript" src="{{ bundlePath }}/appDependencies.bundle.js"></script>
+        <script type="text/javascript" src="{{ bundlePath }}/app.bundle.js"></script>
     </body>
 </html>

--- a/packages/fxa-content-server/server/templates/pages/src/update_firefox.html
+++ b/packages/fxa-content-server/server/templates/pages/src/update_firefox.html
@@ -22,10 +22,7 @@
         <link rel="stylesheet" href="/styles/main.css" />
         <!-- endbuild -->
 
-        <script
-            type="text/javascript"
-            src="{{ bundlePath }}/head.bundle.js"
-        ></script>
+        <script type="text/javascript" src="{{ bundlePath }}/head.bundle.js"></script>
     </head>
     <body class="static">
         <div id="stage">

